From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

v-nagta committed:
Support URL updated

nabhishek committed:
updated ports

v-shils committed:
Merge pull request #35395 from ajlam/release-orcas-ga

Update mysql and pg diagrams, mysql limits

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

hpsin committed:
Editor feedback

hpsin committed:
fix github alias.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-doc-issue-5851

Commit:
Adressing doc issue 5851

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
fix some blocking issues

ajlam committed:
Merge pull request #1 from MicrosoftDocs/release-orcas-ga

Release orcas ga

rjagiewich committed:
Merge pull request #35298 from v-chenyh/data-in

Add back data-in articles

anirudhcavale committed:
Ani Metric Updates for Gartner

ajlam committed:
Add hardware gen availability

anirudhcavale committed:
Updating broken link

anirudhcavale committed:
Updating link

ajlam committed:
fix regions

ajlam committed:
Update regions

snehithm committed:
Metric alert changes

snehithm committed:
Fixing typos

snehithm committed:
Fixing typos

snehithm committed:
Fixing typos

snehithm committed:
Fixing links

snehithm committed:
Fixing links

linda33wj committed:
Add CDS support

ealsur committed:
Updating for 1.3.1 release

1.3.0 was unlisted and replaced by 1.3.1

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

rmca14 committed:
Merge pull request #35499 from ajlam/release-orcas-ga

Add hardware generation region availability

ajlam committed:
Fix typo

Commit:
Merge pull request #35527 from ajlam/patch-1

Fix typo in PG pricing tier

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

hpsin committed:
Acrolinx and feedback

rwike77 committed:
fixed conflict

hpsin committed:
Switch manifest data to codeblocks

anirudhcavale committed:
One More Metric Provider

ChrisGMsft committed:
Remove reference to GetDevices

Removed the note about GetDevicesAsync since it is now deprecated from the SDK

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

nitinme committed:
first wip checkin

Ja-Dunn committed:
Merge pull request #35496 from anirudhcavale/gartner-Ani-metric-Update

Ani Metric Updates for Gartner

nitinme committed:
more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

rolyon committed:
Reader role can't view functions within a function app #5654

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-funcrbac-issue5654

rolyon committed:
Added red box to screenshot, updated article date

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-funcrbac-issue5654

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

snehithm committed:
Fixing metrics taxonomy

snehithm committed:
Fixing metric alert titles

snehithm committed:
Fixing metric alert titles

snehithm committed:
Fixing newer metric alert article

snehithm committed:
Calling old alerts classic metric alerts

snehithm committed:
Fixing broken link

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

nitinme committed:
minor updates

nitinme committed:
another update

nitinme committed:
updating index and toc files

nitinme committed:
updates

rboucher committed:
add additional screen shots

GitHubber17 committed:
Merge pull request #35509 from snehithm/gartner-metric-alerts-update

Metric alert changes

rboucher committed:
updated screenshots

rboucher committed:
updated screenshots

rboucher committed:
Merge branch 'release-monitor-gartner' of https://github.com/MicrosoftDocs/azure-docs-pr into gartSS

Juliako committed:
Update import-and-publish.md

Juliako committed:
Update api-management-howto-protect-backend-with-aad.md

v-nagta committed:
updated

rboucher committed:
updated screenshots

rboucher committed:
updated screenshots

rboucher committed:
updated TOC for screen changes

rboucher committed:
updated TOC for screen changes

rboucher committed:
updated TOC for screen changes

genlin committed:
update the heading

genlin committed:
correct the date

rayne-wiselman committed:
updating article with info requested by customer

Commit:
updates for the rebranding of OMS

CarlRabeler committed:
[sql db] MI - threat detection

nitinme committed:
minor updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

CarlRabeler committed:
acro

rolyon committed:
Repositioned Functions section based on feedback from PM

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-funcrbac-issue5654

ajlam committed:
Merge pull request #2 from MicrosoftDocs/release-orcas-ga

Update fork

rolyon committed:
Updated article date

linda33wj committed:
Add metadata for include file

linda33wj committed:
Tweak metadata

Commit:
updates after review

Commit:
metadata cleanup #4

Commit:
added collector error codes

rboucher committed:
updated bad screenshot

rayne-wiselman committed:
Update vmware-azure-architecture.md

lindavr committed:
adding abstracts to wp list

rboucher committed:
changes for alerts

rayne-wiselman committed:
Update vmware-azure-architecture.md

Commit:
fixed titles

Commit:
Update analysis-services-gateway.md

mumian committed:
[HDI] update spark release notes

bandersmsft committed:
azure policy - updated pricing info

PRMerger18 committed:
Merge pull request #35670 from mumian/0319-spark-releasenote

[HDI] update spark release notes

raiye committed:
Update cloud-services-guestos-msrc-releases.md

dlepow committed:
Batch: Low-pri VMs note

Commit:
Merge pull request #35671 from bandersmsft/master

azure policy - updated pricing info

Commit:
Merge pull request #35629 from Juliako/patch-3

Update api-management-howto-protect-backend-with-aad.md

amanarneja committed:
Fixes MicrosoftDocs/azure-docs#5973

https://review.docs.microsoft.com/azure/network-watcher/ should be  https://docs.microsoft.com/azure/network-watcher/

v-alje committed:
Bringing even with master.

v-shils committed:
Merge pull request #35625 from Juliako/patch-4

Update import-and-publish.md

CarlRabeler committed:
[sql db] 35 TB clarity

v-alje committed:
Bringing even with master.

CarlRabeler committed:
[sql db] always on terminology

CarlRabeler committed:
date

rjagiewich committed:
Merge pull request #35661 from georgewallace/autometadata

metadata cleanup #4

PRMerger19 committed:
Merge pull request #35668 from Minewiskan/patch-2

Update analysis-services-gateway.md

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into actors

rwike77 committed:
addressed reviewer feedback

tynevi committed:
Merge pull request #35662 from lindavr/wpabstracts

[Security] Adding abstracts to wp list

ggailey777 committed:
Fixed typo

Juliako committed:
Update media-services-manage-channels-overview.md

Juliako committed:
Update media-services-manage-channels-overview.md

Juliako committed:
Update media-services-live-streaming-with-onprem-encoders.md

Juliako committed:
Update media-services-manage-live-encoder-enabled-channels.md

raiye committed:
Update cloud-services-guestos-update-matrix.md

rwike77 committed:
minor updates in wording

davidmu1 committed:
reassignment of articles

PRMerger13 committed:
Merge pull request #35673 from raiye/patch-1

Update cloud-services-guestos-msrc-releases.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into v-craic0316

rwike77 committed:
added link

PRMerger15 committed:
Merge pull request #35677 from amanarneja/patch-20

Fixes MicrosoftDocs/azure-docs#5973

rjagiewich committed:
Merge pull request #35596 from rolyon/rolyon-funcrbac-issue5654

[Azure AD RBAC] Reader role can't view functions within a function app #5654

PRMerger19 committed:
Merge pull request #35674 from dlepow/fix2

Batch: Low-pri VMs note

Commit:
Merge pull request #35687 from CarlRabeler/alwayson

Alwayson

Commit:
add sqlops link to the sql db toc

Commit:
remove en-us

PRMerger14 committed:
Merge pull request #35689 from ggailey777/patch-16

Fixed typo

rayne-wiselman committed:
small change for clarity on customer question

curtand committed:
[Azure AD RBAC] update outdated reference to Windows Azure

rwike77 committed:
updating article freshness

neilpeterson committed:
fixes MicrosoftDocs/azure-docs#5901

Ja-Dunn committed:
Merge pull request #35700 from davidmu1/vmrename

reassignment of articles

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

neilpeterson committed:
[AKS] added H2

Ja-Dunn committed:
Merge pull request #35712 from curtand/markus0319

[Azure AD RBAC] update outdated reference to Windows Azure

ajlam committed:
revert data-in docs

raiye committed:
Update cloud-services-guestos-update-matrix.md

rwike77 committed:
fixed link

ajlam committed:
Update date

linda33wj committed:
Trigger rebuild

ajlam committed:
Update article dates

rjagiewich committed:
Merge pull request #35704 from stevestein/sql-db-add-sqlops

Sql db add sqlops

PRMerger14 committed:
Merge pull request #35694 from Juliako/patch-4

Update media-services-live-streaming-with-onprem-encoders.md

rboucher committed:
Update for recent release

Juliako committed:
Update media-services-manage-channels-overview.md

rboucher committed:
fix acrolinx

rboucher committed:
fix acrolinx

rboucher committed:
small fix

rboucher committed:
small fix

rwike77 committed:
Supported versions for overriding endpoints.  

Addresses https://github.com/MicrosoftDocs/azure-docs/issues/5505.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

nitinme committed:
minor update

zhiweiwangmsft committed:
Minor: Fix typo

fix typo

Commit:
Merge pull request #35695 from Juliako/patch-5

Update media-services-manage-live-encoder-enabled-channels.md

Commit:
Merge pull request #35729 from Juliako/patch-4

Update media-services-manage-channels-overview.md

Commit:
Merge pull request #35400 from v-nagta/cylancev1

USER STORY 1209124 SaaS App Tutorial: CylancePROTECT Update

bwren committed:
SQL, Web Apps, Capacity

bwren committed:
update

GitHubber17 committed:
Merge pull request #35512 from linda33wj/master

Add CDS support in ADF

GitHubber17 committed:
Merge pull request #35016 from hpsin/OptionalClaimsMAster

Add Optional Claims documentation

GitHubber17 committed:
Merge pull request #35720 from ajlam/release-orcas-ga

Revert data-in docs

tynevi committed:
Merge pull request #35690 from tysonn/linda

Linda issue

ajlam committed:
Update CLI extension version

ajlam committed:
update pg cli extension version

PRMerger20 committed:
Merge pull request #35696 from raiye/patch-2

Update cloud-services-guestos-update-matrix.md

GitHubber17 committed:
Merge pull request #35593 from nitinme/nitinme-databricks-sqldw-tutorial

Databricks + SQL DW tutorial

GitHubber17 committed:
Merge pull request #35722 from ajlam/patch-2

Update date

GitHubber17 committed:
Merge pull request #35746 from ajlam/patch-3

Update CLI extension version

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-sqldw-tutorial

nitinme committed:
minor fixes

Commit:
Merge pull request #35403 from nabhishek/patch-20

updated ports

SnehaGunda committed:
removing note

PRMerger14 committed:
Merge pull request #35663 from ruturaj/colerrors

added collector error codes

PRMerger19 committed:
Merge pull request #35736 from zhiweiwangmsft/patch-21

Minor: Fix typo

PRMerger17 committed:
Merge pull request #35751 from SnehaGunda/master

removing note

genlin committed:
update the link

genlin committed:
update the date

genlin committed:
revert the changes

genlin committed:
revert change

Commit:
Merge pull request #35692 from Juliako/patch-3

Update media-services-manage-channels-overview.md

genlin committed:
add reference to a link issue

PRMerger13 committed:
Merge pull request #35734 from rwike77/patch-36

Supported versions for overriding endpoints.

PRMerger15 committed:
Merge pull request #35481 from MikkelHegn/fix-doc-issue-5851

Fix doc issue 5851

PRMerger18 committed:
Merge pull request #35697 from rwike77/setup

minor updates in wording

PRMerger19 committed:
Merge pull request #35713 from rwike77/freshness

updating article freshness

genlin committed:
update the link

Duncanma committed:
Merge pull request #35147 from MicrosoftDocs/HeadingFix

Fixing heading style, for both upcoming validation rules

rboucher committed:
small fix to regions

Commit:
Merge pull request #35753 from genlin/master304

boot-diagnostics.md

Commit:
Merge pull request #35728 from rboucher/govupdate

Update government difference for recent release of Alerts

rayne-wiselman committed:
Update vmware-azure-architecture.md

rayne-wiselman committed:
removing VHD reqs from vmware support matrix

rayne-wiselman committed:
Update vmware-azure-architecture.md

PRMerger14 committed:
Merge pull request #35711 from rayne-wiselman/rayne-march19-2

small change for clarity on customer question

rayne-wiselman committed:
Update vmware-azure-architecture.md

ajaycode committed:
Link to overview page

PRMerger14 committed:
Merge pull request #35759 from bwren/kusto-ga-4

Kusto GA - SQL, Web Apps, Capacity

Commit:
Merge pull request #35757 from ajaycode/patch-12

Link to overview page

Commit:
Added required actions to question 1 answer.

Commit:
Fixed typo

PRMerger16 committed:
Merge pull request #35761 from jimdial/patch-97

Added required actions to question 1 answer.

Commit:
updates for the rebranding of OMS

rothja committed:
Changes per CSS feedback

Commit:
updates for the rebranding of OMS

Commit:
Merge pull request #35684 from CarlRabeler/MI

[sql db] 35 TB clarity

Commit:
Merge pull request #35750 from nitinme/nitinme-databricks-sqldw-tutorial

Incorporating feedback

itechedit committed:
Merge pull request #35676 from rboucher/dukeupdate2

changes for alerts

christiankuhtz committed:
remove classic

ehrlinger committed:
Update the PM github link.

PRMerger10 committed:
Merge pull request #35766 from rothja/miscupdates

Changes per CSS feedback

Commit:
fix wording one file

christiankuhtz committed:
add xref to outbound arm

christiankuhtz committed:
updated date

christiankuhtz committed:
update date

charwen committed:
Updated CenturyLink

rmca14 committed:
Merge pull request #35627 from rboucher/gartSS

Screen shot updates

PRMerger14 committed:
Merge pull request #35769 from ehrlinger/master

Update the PM github link.

PRMerger15 committed:
Merge pull request #35639 from craigcaseyMSFT/v-craic0316

updates for the rebranding of OMS

Commit:
fix wording on a file

PRMerger17 committed:
Merge pull request #35772 from charwen/patch-46

Updated CenturyLink

christiankuhtz committed:
bug fix

PRMerger20 committed:
Merge pull request #35765 from craigcaseyMSFT/v-craic0319

updates for the rebranding of OMS

Commit:
Merge pull request #35515 from ealsur/master

Updating for Cosmos DB Change Feed SDK 1.3.1 release

Commit:
Merge pull request #35716 from neilpeterson/open-faas-pathc

[AKS] added H2

Commit:
Merge pull request #35714 from neilpeterson/aks-github-issue-5901

fixes MicrosoftDocs/azure-docs#5901

rmca14 committed:
Merge pull request #35002 from rwike77/actors

Actors refactoring

Commit:
Merge pull request #35771 from christiankuhtz/patch-84

add xref to outbound arm

Commit:
Merge pull request #35768 from christiankuhtz/patch-83

remove classic

Commit:
ASDK udpates

v-alje committed:
Merge pull request #35715 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger14 committed:
Merge pull request #35775 from jeffgilb/asdk-updates

ASDK udpates

v-alje committed:
Merge pull request #35591 from MicrosoftDocs/release-monitor-gartner

Release monitor gartner

PRMerger15 committed:
Merge pull request #35581 from ChrisGMsft/patch-19

Remove reference to GetDevices

barclayn committed:
deleting the azlog asc article

v-alje committed:
Merge pull request #35778 from MicrosoftDocs/FromPublicRepo

From public repo

douglaslMS committed:
Made casing fixes requested by Databricks.

tynevi committed:
Merge pull request #35780 from barclayn/delete-azlog-asc

deleting the azlog asc article

Commit:
Merge pull request #35783 from douglaslMS/adf-databricks-casing

Made casing fixes requested by Databricks.

v-alje committed:
Merge pull request #35784 from MicrosoftDocs/release-orcas-ga

Release orcas ga

rjagiewich committed:
Merge pull request #35645 from CarlRabeler/mi-audit

[sql db] MI - threat detection

Lisaco88 committed:
Merge pull request #35635 from rayne-wiselman/rayne-march19

updating article with info requested by customer
